apps/tenant-management-webapp/src/app/pages/admin/services/form/definitions/formStepper.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { useState } from 'react';
+import { GoAFormStepper, GoAFormStep, GoAPages, GoAButton } from '@abgov/react-components-new';
+import { materialRenderers, materialCells } from '@jsonforms/material-renderers';
+import { JsonForms } from '@jsonforms/react';
+import { Grid, GridItem } from '@components/Grid';
+import { categorizationRendererTester } from './addEditFormDefinitionEditor';
+import FormStepperControl from './FormStepperControl';
+
+import { ReviewItem } from './style-components';
+
+interface StepperProps {
+  uiSchema: Record<string, unknown>;
+  dataSchema: Record<string, unknown>;
+  data: Record<string, unknown>;
+}
+
+declare type Status = 'complete' | 'incomplete';
+
+export const FormStepper: React.FC<StepperProps> = ({ uiSchema, data, dataSchema }) => {
+  const [step, setStep] = useState<number>(-1);
+  const [stepData, setStepData] = useState(data);
+
+  function setPage(page) {
+    if (page < 1 || page > uiSchema.elements?.length + 1) return;
+    setStep(page);
+  }
+
+  const renderers = [
+    ...materialRenderers,
```

Comment:
You're not really making use of materialRenderers here; I mean, couldn't you just as well use vanillaRenderers?  I don't really know if it makes much difference except that material-ui is a large package and if it is unnecessary why include it?

Reply:
I think it's for elements that we don't have a wrapper for, but I could try without it

Reply:
That's true, but I think vanillaRenderers will do the trick.